.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+exclude=alembic/versions,.venv
+docstring-convention=google
+max-line-length = 110
```

Comment:
Why 110?

Reply:
I think `black` defaults to 88: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length